release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -406,6 +406,21 @@ With this release, the etcd tuning parameters can be set to values that optimize
 * `Standard`
 * `Slower`
 
+[discrete]
+[id="ocp-4-16-unauthenticated-users-cluster-role-bindings_{context}"]
+=== Unauthenticated users were removed from some cluster role bindings.
+
+In {product-title} {product-version} unauthenticated users will no longer have access to most cluster role bindings. Changing this access for unathenticated users adds an additional layer of security and will not impact most users. This change is for new clusters, previous clusters will not be effected.
+
+There are use cases where allowing unauthenticated users specific cluster role bindings is desired. For example, the `system:webhook` cluster role allows users to trigger builds from external systems such as GitHub, GitLab, and Bitbucket. Unauthenticated users can still do this if granted access to the `system:webhook` cluster role binding by a cluster administrator.
```

Comment:
Maybe reorder the last sentence to be more active voice - start with saying that cluster admins can grant access...